tests/test_evaluation/test_rank_based_metrics.py

```diff
@@ -49,6 +57,14 @@ class ZArithmeticMeanRankTests(cases.RankBasedMetricTestCase):
 
     cls = pykeen.metrics.ranking.ZArithmeticMeanRank
 
+    def test_adjustment(self):
+        """Check that the adjustment happened."""
+        base_instance = pykeen.metrics.ranking.ArithmeticMeanRank()
+        self.assertNotEqual(
+            self.instance(ranks=self.ranks, num_candidates=self.num_candidates),
+            -base_instance(ranks=self.ranks, num_candidates=self.num_candidates),
```

Comment:
why is it negative for this one and not the others? because it's an `IncreasingZMixin` and not a `DecreasingZMixin`? Can we encode logic for that in the test?

Reply:
Yes, exactly. we could introduce another base test case, but there is only one metric which is affected :thinking: